CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 This file is used to list changes made in each version of the nginx cookbook.
 
+## 10.0.0-dev (2019-04-09)
```

Comment:
Given this isn't released yet, I'd propose these live under the `Unreleased` heading. And then moved to a `10.0.0` when released